### ISSUE TITLE
logging setup via customize

### ISSUE THREAD

**Issue by zabbal** [jorgenschaefer/circe#109]
According to documentation at https://github.com/jorgenschaefer/circe/wiki/Configuration logging should be enabled by setting circe-channel-mode-hook to enable-lui-logging

However with version 20150208.1020 of circe there is no such hooks visible in Customize facility. Also it's unclear where exactly the logs would be saved into. The $HOME/.local/logs seems like a sensible default but most people would prefer to have it easily configurable via Customize.

In summary - would be nice to have an ability to enable logging via simple customize variable with optional ability to configure destination.

**Comment by jorgenschaefer** [comment]
Thank you for the report!
The following code in your .emacs will enable logging:

```
(load "lui-logging" nil t)
(enable-lui-logging-globally)
```

You are right, though, that the configuration should be updated and it would be nice if this was available via customize.

**Comment by zabbal** [comment]
Thanks for quick response. What about the destination? Where will (enable-lui-logging-globally) save logs into and how do I alter it?

**Comment by jorgenschaefer** [comment]
Ah, sorry, missed that – logs will go to `lui-logging-directory`, formatted according to `lui-logging-file-format`. By default, that looks like `~/.logs/#emacs_2015-02-09.txt`

**Comment by jorgenschaefer** [comment]
Fixed the Configuration page.

I'm unsure if this is easy to do via customize, or if it really would add so much if this was possible. I'll leave that alone for now.

As for the location, I _think_ the correct location would be `~/.local/share/circe/logs/`, but I'm not convinced that's really preferable over `~/.logs/`, so I'll leave the default as is for now.

Please do not hesitate to comment if you disagree, or create further issues if you encounter more problems or questions :-)
